### PR TITLE
Typography list

### PR DESCRIPTION
**Pull request by varun-Tad**
I have created a few Typography and list components.

### CODE REVIEW

File: Components/Typography/typography.css [hunk 1]
```diff
@@ -34,24 +35,67 @@
...
+.normal-text{
+    font-weight: 400;
 }
 
-.font-extra-small{
-    font-size: 1rem;
+.light-text{
+    font-weight: 100;
+}
+
```
Comment (Naman-Saxena1) [inline Components/Typography/typography.css:91]: Your normal and light text looks same, I believe font-weight is not working properly for some reason.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Everything else looks good to me 👍

File: documentation.html [hunk 2]
```diff
@@ -536,6 +538,125 @@ <h2 class="component-sub-heading">Examples of input components</h2>
...
+
+    <div class="lists">
+        <ol class="lowercase-letter">
+            <li>Item one</li>
+            <li>Item two</li>
+            <li>Item three</li>
+        </ol>
+    </div>
+
+  <h2 class="component-sub-heading">Ordered Lists</h2>
```
Comment (sri-rishi) [inline documentation.html]: Heading  should be Unordered lists

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Other things looks fine to me

File: Components/Typography/typography.css (hunk 1 above)
Comment (varun-Tad) [reply Components/Typography/typography.css:91]: I played with font-weight specifically using normal, lighter and 100 using playit feature in w3schools, and in all these 3 cases the text looks the same to me too. I don't know if it's just me who's not able to notice the 
difference. @Naman-Saxena1

File: documentation.html (hunk 2 above)
Comment (varun-Tad) [reply documentation.html]: Noted. Thanks 👍

**Pull request merged**